mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (result) {
+      return res.status(404).json({ message: "Order not located" });
+    }
+    res.status(200).json({ message: "Order eradicated successfully" });
```

Comment:
In the `purgeOrder` function, the success response message is inconsistent with the other functions. It should be standardized to follow a consistent format. For example, it could be `res.json({ success: true, message: "Order deleted successfully" });`.